Source/DafnyCore/Compilers/Compiler-python.cs

```diff
@@ -843,11 +855,11 @@ private class ClassWriter : IClassWriter {
       var wStmts = wr.Fork();
       wr.Write($"raise {DafnyRuntimeModule}.HaltException(");
       if (tok != null) {
-        wr.Write($"{DafnyRuntimeModule}.Seq(\"{Dafny.ErrorReporter.TokenToString(tok)}: \") + ");
+        wr.Write($"\"{ErrorReporter.TokenToString(tok)}: \" + {DafnyRuntimeModule}.string_of(");
```

Comment:
Just curious, is this an optimization or fixing a bug of some kind?

Reply:
Both. Before we used `+` on a `str` and an arbitrary python object to create a string `Seq`. This would only be used when catching the exception, so no need for the dafny wrapper. Instead of fixing this mess, I got rid of it.